pkgs/development/php-packages/scrypt/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildPecl, lib, fetchFromGitHub }:
+
+let
+  version = "v2.0.1";
+  revision =  "845b889bdbe817afe1633237f8fc68667c7a700b";
+in buildPecl {
+  inherit version;
+
+    pname = "scrypt";
+
+    src = fetchFromGitHub {
+      owner = "DomBlack";
+      repo = "php-scrypt";
+      rev = revision;
+      sha256 = "sha256-JUA8Do43Gmtc0ZWJHBtBzSQ+AhE5nGkiRg8gkVF+Cec=";
+    };
+
+    doCheck = true;
+    checkTarget = "test";
+
+    zendExtension = false;
```

Comment:
This should be the default, no?

https://github.com/NixOS/nixpkgs/blob/8d80108d6d7b1c938f46a41700861a65c599f4a1/pkgs/development/interpreters/php/generic.nix#L126